pkg/model/manifest.go

```diff
@@ -717,13 +717,15 @@ func (m *Manifest) setDefaults() error {
 		}
 	}
 
-	for _, b := range m.Build {
+	for k, b := range m.Build {
 		if b.Name != "" {
 			b.Context = b.Name
 			b.Name = ""
 		}
-		if !(b.Image != "" && len(b.VolumesToInclude) > 0 && b.Dockerfile == "") {
```

Comment:
@pchico83 @jLopezbarb  Don't we need this check anymore?

Reply:
Now it's done at serialisation time instead, so we don't have to check this anymore